src/content/contributing/adding-wallet/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Adding wallets
+description: The policy we use when adding a wallet to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding wallets {#adding-wallets}
+
+We want to make sure we show a variety of wallets covering the feature rich landscape of wallets so that users can navigate Ethereum in a confident manner.
```

Comment:
```suggestion
We want to make sure we show a variety of wallets covering the feature-rich landscape of wallets so that users can navigate Ethereum in a confident manner.
```